src/annotator/sidebar.js

```diff
@@ -24,14 +24,130 @@ const defaultConfig = {
   },
 };
 
-export default class Sidebar extends Host {
+/**
+ * Create the JSON-serializable subset of annotator configuration that should
+ * be passed to the sidebar application.
+ */
+function createSidebarConfig(config) {
```

Comment:
It's still a little bit intertwined with the details of the rest of the annotator code at the moment. See the reference to the `pluginClasses` property which is set in `annotator/index.js`. This may make sense to do in future though.